### PR TITLE
Resolve broken build issues

### LINKED ISSUE #27: Fix Broken Jenkins Builds
The CI jobs are running against the branch `hello-world`. Building against `master` fails the build ([log](http://jenkins.dev.cwds.io:8080/view/cm/job/cm-stable/2/console)).

Fix the build plz

### PR DESCRIPTION
**Pull request by eckdanny-osi**
resolves #27 

Root issue was all the extra opts that `docker-workflow` plugin basses to `docker run ...` (e.g.; workspace, bind-mounts, etc.)

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,98 +1,60 @@
...
-  slackSend channel: "#casemanagement-stream", baseUrl: 'https://hooks.slack.com/services/', tokenCredentialId: 'slackmessagetpt2', color: colorCode, message: summary
-  emailext(
-      subject: subject,
-      body: details,
-      attachLog: true,
-      recipientProviders: [[$class: 'DevelopersRecipientProvider']],
-      to: "ramu.kammagani@osi.ca.gov"
+DOCKER_GROUP = 'cwds'
+DOCKER_IMAGE = 'casemanagement'
+DOCKER_REGISTRY_CREDENTIALS_ID = '6ba8d05c-ca13-4818-8329-15d41a089ec0'
```
Comment (edgibbs) [inline Jenkinsfile]: This isn't new, but are we opening up anything with storing these in the Jenkinsfile?

File: Jenkinsfile [hunk 2]
```diff
@@ -1,98 +1,60 @@
...
+SLACK_CHANNEL = '#casemanagement-stream'
+SLACK_CREDENTIALS_ID = 'slackmessagetpt2'
```
Comment (edgibbs) [inline Jenkinsfile]: Same question here, if we should store credentials.

File: Jenkinsfile [hunk 3]
```diff
@@ -1,98 +1,60 @@
...
+            app = docker.build("${DOCKER_GROUP}/${DOCKER_IMAGE}:${env.BUILD_ID}")
         }
-    }
-    stage('Run application') {
-
-        sh 'docker stop casemanagement-test || exit 0; docker rm casemanagement-test || exit 0'
-        withDockerRegistry([credentialsId: '6ba8d05c-ca13-4818-8329-15d41a089ec0']) {
-          sh 'docker pull cwds/casemanagement '
-          sh 'docker run -d -p 8888:3000 --name casemanagement-test -e APP_NAME=casemanagement cwds/casemanagement'
+        app.withRun("-e CI=true") { cm ->
```
Comment (edgibbs) [inline Jenkinsfile]: So curious as to what -e CI=true does here, and can we use something like caseManagementImage over 'cm' here.  Yes, I'm OCD about full words in variable names.

Comment (eckdanny-osi) [reply Jenkinsfile]: `-e CI=true` is for [`jest` config](https://facebook.github.io/jest/docs/en/cli.html#ci)

Comment (eckdanny-osi) [reply Jenkinsfile]: updated arg name in the lambda in 72cd601

File: Jenkinsfile (hunk 1 above)
Comment (eckdanny-osi) [reply Jenkinsfile]: yes not new. I assumed these are just _keys_ to credentials that Jenkins maintains opaquely. I'll confirm that...

**Pull request merged**